### DISCUSSION TITLE
[Scheduler] Is it possible to schedule a message once for a specific date + time?

### DISCUSSION THREAD

**Discussion by ruudk** [symfony/symfony#58229] [Q&A]
In the [documentation](https://symfony.com/doc/current/scheduler.html) I see it talks about recurring all the time.

Would it be possible to schedule a message **once**, in the future, at a specific date + time?

Or would the best way to deal with this be a custom TriggerInterface?

**Comment by MrYamous** [comment]
I think this might be achieved using `from` and `until` options or [ExcludeTimeTrigger](https://github.com/symfony/symfony/blob/7.1/src/Symfony/Component/Scheduler/Trigger/ExcludeTimeTrigger.php)
